build.yaml

```diff
@@ -52,6 +55,17 @@ steps:
    dependsOn:
     - default_ns
     - batch_pods_ns
+ - kind: deploy
+   name: deploy_ci_sa
+   namespace:
+     valueFrom: batch_pods_ns.name
+   config: ci/service-account.yaml
+   dependsOn:
+    - default_ns
+    - batch_pods_ns
+   scopes:
+    - test
+    - dev
```

Comment:
This is the actual ci-agent configuration, right?  In which case, this can be unconditional so ci-agent is managed by build.yaml now, right?

Reply:
I didn't want to overwrite ci-agent in the batch-pods namespace until I knew what permissions it had. I can't figure out how to get the yaml configuration using kubectl and I also looked at the yaml configuration on the broad server.

Reply:
Ah, yeah, you can't because ci-agent has cluster-wide admin bindings because it needs to be able to create and destroy namespaces.

There are two kinds of roles and role binding: cluster wide and namespace local.  ci-agent has a cluster-wide role:

```
$ kubectl get -o yaml clusterrolebinding ci-agent-cluster-admin
apiVersion: rbac.authorization.k8s.io/v1
kind: ClusterRoleBinding
metadata:
  name: ci-agent-cluster-admin
roleRef:
  apiGroup: rbac.authorization.k8s.io
  kind: ClusterRole
  name: cluster-admin
subjects:
- kind: ServiceAccount
  name: ci-agent
  namespace: batch-pods
```

Reply:
My suggestion was overly ambitious, disregard.